receipt_agent/receipt_agent/subagents/place_finder/backfill_receipt_place.py

```
@@ -28,24 +28,29 @@
 - Phase 4: Mark migration complete when all receipts have ReceiptPlace
 """
 
+import argparse
 import asyncio
 import logging
 import sys
 from dataclasses import dataclass, field
 from datetime import datetime, timezone
 from typing import Any, List, Optional
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)